blog/fine-grained-news-2024-08.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Fine Grained News - August 2024
+description: Fine Grained News
+slug: fine-grained-news-2024-08
+date: 2024-08-30
+authors: hello-caleb
+tags: [newsletter]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# Fine Grained News
+
+Welcome to the August 2024 edition of Fine Grained News! We are excited to bring you the latest updates, features, and community highlights from OpenFGA. 
+
+## Just Shipped!
+
+* **OpenFGA v1.6.0:** The [latest OpenFGA release](https://github.com/openfga/openfga/releases/tag/v1.6.0) enables support for [query consistency options](https://openfga.dev/docs/interacting/consistency) and included additional performance enhancements.
```

Comment:
```suggestion
* **OpenFGA v1.6.0:** The [latest OpenFGA release](https://github.com/openfga/openfga/releases/tag/v1.6.0) enables support for [query consistency options](https://openfga.dev/docs/interacting/consistency) 
```

we didn't just enabled support, we made it the default: https://github.com/openfga/openfga/commit/15a2a97048fd981abd55bf98b57816f616e99080

Reply:
@miparnisari yeah, but I think we never shipped a release where you could turn them on after v1.5.9

Reply:
Thanks for the clarification, @miparnisari. Should this be clarified, @aaguiarz?

Reply:
I changed "included" for "includes", and I think that's fine. OpenFGA 1.6 is already released so we can publish this now.

Reply:
Thanks for the input